model-optimizer/mo/utils/versions_checker.py

```diff
@@ -61,7 +61,11 @@ def parse_and_filter_versions_list(required_fw_versions, version_list, env_setup
     # check environment marker
     if len(splited_requirement) > 1:
         env_req = splited_requirement[1]
-        splited_env_req = re.split(r"==|>=|<=|>|<|~=", env_req)
+        if any([x in splited_requirement[1] for x in [' and ', ' or ']]):
+            log.error("The version checker doesn't support environment marker combination and it will be ignored: {}"
+                      "".format(splited_requirement[1]), extra={'is_warning': True})
+            return version_list
+        splited_env_req = re.split(r"==|>=|<=|>|<|~=|!=", env_req)
```

Comment:
nit: Change everywhere "splited" to "split".  Refer to https://en.wiktionary.org/wiki/splitted

Reply:
Ok, done.